source/seedsource/serializers.py

```diff
@@ -74,3 +74,10 @@ class GeneratePDFSerializer(serializers.Serializer):
     tile_layers = serializers.JSONField()
     zoom = serializers.IntegerField()
     opacity = serializers.FloatField()
+
+
+class RegionSerializer(serializers.ModelSerializer):
+
```

Comment:
The difference is that there the blank line separates the ```class Meta``` declaration from items at the same indent level, whereas here, there is nothing at the same level as the ```Meta``` class definition; the indent separates it from the ```RegionSerializer``` definition above.

That said, spacing is a personal preference. If it looks better to you with the extra space, I'm fine with that.